.github/workflows/azure-site-extension.yml

```diff
@@ -0,0 +1,71 @@
+name: Azure Site Extension
+
+on:
+  workflow_dispatch:
+
+env:
+  SPEC_FILE_TEMPLATE: 'NewRelic.Azure.WebSites.Extension.NodeAgent.nuspec'
+
+jobs:
+  create_extension_bundle:
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        node-version: ['lts/*']
+        arch: [ x64 ]
```

Comment:
we're only running this on x64 but the nuspec includes this in the name. is this necessary? are we going to document what doesn't work on 32 bit?

Reply:
I'm in favor of documenting the difference *and* removing the architecture from the nuspec name. I'll update.

Reply:
I've restored the x86 build to this, and added some documentation on what doesn't work on 32. (If we supply x86 builds, CLM works as expected on Node 20 for win 32, but not for Node 18.)